componentapiexample/src/main/java/net/gini/android/vision/component/GiniVisionAppCompatActivity.java

```diff
@@ -1,11 +1,15 @@
 package net.gini.android.vision.component;
```

Comment:
There is a lot of duplicated code in this file if you compare it with `GiniVisionActivity`. Is there a way to join them  and get rid of the duplicated code?

Reply:
Clients will have to study either one or the other depending on whether they use the support library or not. I would not abstract the code away for the example. I would like to keep everything in one file for each case. They share logic but shouldn't share code. Presenting an independent example for each case is more important than maintainability.